_includes/video/pbs-jw01.html

```diff
@@ -3,8 +3,7 @@
 
 <head>
 {% include head--common.html %}
-    <!--production version of prebid.js-->
-    <script async src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
+    {% include prebidjs-non-prod.html %}
     <script>
```

Comment:
no CMP tags

Reply:
This code does nothing as long the relevant js is loaded (prebid.js and player).
Only callbacks to a que are pushed